### DISCUSSION TITLE
Fail to launch the app but it works well on Geforce experience

### DISCUSSION THREAD

**Discussion by riskeay** [LizardByte/community#386] [Sunshine]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your issue described in the documentation?

- [X] I have read the documentation

### Is your issue present in the nightly release?

- [X] This issue is present in the nightly release

### Describe the Bug

can't open the Geshin impact and steam, but can open the desktop

https://github.com/LizardByte/Sunshine/assets/95848505/5c432f60-afcd-418f-9edf-934b514fffce

### Host Operating System

Windows

### Operating System Version

Windows 10

### Architecture

64 bit

### Sunshine commit or version

0.21.0

### Package

Windows - installer

### GPU Type

AMD

### GPU Model

NVIDIA GeForce GTX 1070 Ti

### GPU Driver/Mesa Version

551.23

### Config

```Shell
UPnP = enabled
```

### Apps

```json
{
    "env": {
        "PATH": "$(PATH);$(ProgramFiles(x86))\\Steam"
    },
    "apps": [
        {
            "name": "Desktop",
            "image-path": "desktop.png"
        },
        {
            "name": "Steam Big Picture",
            "detached": [
                "steam steam:\/\/open\/bigpicture"
            ],
            "image-path": "steam.png"
        },
        {
            "name": "YuanShen",
            "output": "",
            "cmd": "D:\\Genshin Impact\\Genshin Impact Game\\YuanShen.exe",
            "exclude-global-prep-cmd": "false",
            "elevated": "true",
            "auto-detach": "true",
            "image-path": ""
        }
    ]
}
```

### Relevant log output

```shell
[2024:02:03:13:54:06]: Info: // Ignore any errors mentioned above, they are not relevant. //
[2024:02:03:13:54:06]: Info: 
[2024:02:03:13:54:06]: Info: Found H.264 encoder: h264_nvenc [nvenc]
[2024:02:03:13:54:06]: Info: Found HEVC encoder: hevc_nvenc [nvenc]
[2024:02:03:13:54:06]: Info: Executing: [D:\Genshin Impact\Genshin Impact Game\YuanShen.exe] in ["D:\"]
```

**Comment by ReenigneArcher** [comment]
Firstly, you need to quote your command since you spaces in the path. Surround the entire command in double quotes.

Second of all,, you should not run it elevated, unless absolutely necessary!

**Reply by riskeay** [reply]
I change it as you said, but the situatuion is still there.
{
"name": "YuanShen",
"output": "",
"cmd": ""D:\Genshin Impact\Genshin Impact Game\YuanShen.exe"",
"exclude-global-prep-cmd": "false",
"elevated": "false",
"auto-detach": "true",
"image-path": ""
}

**Reply by ReenigneArcher** [reply]
You need to do this in the web ui, not manually. Now you have invalid json...

**Reply by riskeay** [reply]
I change the web ui as wiki says, still doesn't work.
![图片](https://github.com/LizardByte/Sunshine/assets/95848505/74f522de-844b-457c-81dc-c4710067b0ad)
![图片](https://github.com/LizardByte/Sunshine/assets/95848505/4a97c2bd-a9d7-498f-b75a-b8bd889778a5)

**Reply by ReenigneArcher** [reply]
You need to surround the working directory with quotes...

**Reply by riskeay** [reply]
![图片](https://github.com/LizardByte/Sunshine/assets/95848505/2a8383b0-dc88-4b8c-993c-c088b26388fa)
not working either.

**Reply by riskeay** [reply]
and the json file becomes this 
![图片](https://github.com/LizardByte/Sunshine/assets/95848505/09ff88eb-98ef-4a8e-bb9c-2555b108091a)

[a comment by an automated account is not shown]
